pkg/apis/elasticsearch/v1alpha1/types.go

```diff
@@ -56,6 +56,31 @@ type ElasticsearchNode struct {
 	Storage      ElasticsearchNodeStorageSource `json:"storage"`
 }
 
+type ElasticsearchStorageSpec struct {
+	HostPath *ElasticsearchHostPathSpec `json:"hostPath,omitempty"`
```

Comment:
We can update it to use `localvolume` under the covers if we want. That's why I have it as a new struct. They have similar required fields

Reply:
Looking at the [documentation](https://docs.openshift.com/container-platform/3.11/install_config/configuring_local.html#local-volume-using-raw-block-device-pv) I don't believe there is a need to have 'hostpath' since everything is configured via a PVC.  We may need to expand the info required for us to properly build the PVC spec since there is mention of 'volumemode=block' which I think is what we would choose.

Reply:
Adding comments here that were originally posted to slack:
```
jcantril: we are not supporting host path. It’s my understanding it is deprecated in favor of local volume. We need to take this opportunity to cut our loses and start fresh. This is a place where we may need to specify some precondition and steps that need to be undertaken before we will adopt a cluster
ewolintez: im using `hostpath` and `localvolume` interchangeably there (incorrectly) since they are the same for the most part in terms of what we specify to create it
My point is with a PVC there is no need to define any path so why is this even an option
jcantril: My point is with a PVC there is no need to define any path so why is this even an option. 
 All the local volume set up is outside the control of the operator
```
In answer to your question `... would [localvolume] that be available to us as a storage class...`: yes, as described here: https://docs.openshift.com/container-platform/3.11/install_config/configuring_local.html#local-volume-using-raw-block-device-pv

The documentation details how to setup localvolumes which is heavily contingent on an admin setting up the volumes, selinux labels, and storageclass.  All my understanding here is this would be outside of what the `cluster-logging-operator` is allowed to do which is fine with me; it simplifies our operator.

Reply:
i'll remove the `localvolume` option then... existing PVC or storageclass will be the way to go